src/__tests__/planning.test.ts

```diff
@@ -25,8 +30,8 @@ describe("plan", () => {
 
     expect(xyMotions(p)).toEqual([
       {from: {x: 0, y: 0}, to: {x: 10, y: 10}, penPos: 0},
-      {from: {x: 10, y: 10}, to: {x: 10, y: 10}, penPos: AxidrawFast.penDownPos},
-      {from: {x: 10, y: 10}, to: {x: 0, y: 0}, penPos: device.penPctToPos(0)},
+      {from: {x: 10, y: 10}, to: {x: 10, y: 10}, penPos: positions.up},
```

Comment:
I think this is a typo, because `penDownPos` became `positions.up` instead of `positions.down`?